ivy_tests/test_ivy/test_frontends/test_numpy/test_manipulation_routines/test_changing_kind_of_array.py

```diff
@@ -0,0 +1,38 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# asanyarray
+
+@handle_frontend_test(
+    fn_tree="numpy.asanyarray",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("valid")),
```

Comment:
See [this decorator example](https://github.com/unifyai/ivy/blob/abb765da1cb2b7b14de03220e06061c5efd7d14b/ivy_tests/test_ivy/test_frontends/test_numpy/test_linear_algebra/test_matrix_and_vector_products.py#L134). All of the inputs to the function are defined with strategies i.e. `n` in this case. This also needs to be done here. Currently the inputs `order` and `like` to `test_numpy_asanyarray` are not defined.

Reply:
I tried committing and pushing but it couldn't. I also tried to create another pull request based on what Mr. Dan posted on Ivy discord. Please what should I do? Thanks for being patient with me.